ui/src/app/views/account-creation/component.ts

```diff
@@ -101,6 +90,34 @@ export class AccountCreationComponent {
     });
   }
 
+  get showPasswordsDoNotMatchError() {
+    // We do not want to show errors if nothing is typed yet. This is caught by the required
+    // fields case.
+    if (this.password === undefined || this.passwordAgain === undefined
+      || this.password.length === 0 || this.passwordAgain.length === 0) {
+        return false;
+    }
+    return !(this.password === this.passwordAgain);
+  }
+
+  get showPasswordLengthError() {
+    // We do not want to show errors if nothing is typed yet. This is caught by the required
```

Comment:
I don't think I like the UX of live password validation with negative feedback. The annoyance here is that even if I have a valid password in mind, the UI unjustly complains at me halfway through typing it.

If we're going to do live validation, I think the better UX would be to use positive feedback on satisfying requirements, e.g. pw requirements are listed in bullet form, and you get a checkmark once you satisfy each.

Happy to defer to UX on this, it's just kind of a pet peeve of mine in web UIs.

Reply:
Tagging @fknabe in since he filed the ticket.

Reply:
That said, for feedback purposes, I think I agree that positive feedback would be better.

Reply:
I'm really glad CH brought this up - it is a general concept we should keep in mind for all of our UIs.

Reply:
Also going to tag @lstrano-broad for his thoughts.

Reply:
Agree CH, so let's acknowledge any positive/error feedback once the user goes to the next field - that would assume the user is done typing.

Reply:
@lstrano-broad lets meet in person to go through this?

Reply:
Sure

Reply:
Have met with Lou, closing this out for favor of more work to be done. Should have a new PR up soonish.